mantidimaging/core/rotation/polyfit_correlation.py

```diff
@@ -26,22 +25,40 @@ def do_calculate_correlation_err(store: np.ndarray, search_index: int, p0_and_18
 
 
 def find_center(images: ImageStack, progress: Progress) -> tuple[ScalarCoR, Degrees]:
-    # assume the ROI is the full image, i.e. the slices are ALL rows of the image
+    if images is None or images.proj180deg is None:
+        raise ValueError("images and images.proj180deg cannot be None")
+    # Assume the ROI is the full image, i.e. the slices are ALL rows of the image
     slices = np.arange(images.height)
-    shift = pu.create_array((images.height, ))
-
+    shift = pu.create_array((images.height, ), dtype=np.float32)
     search_range = get_search_range(images.width)
-    min_correlation_error = pu.create_array((len(search_range), images.height))
-    shared_search_range = pu.create_array((len(search_range), ), dtype=np.int32)
-    shared_search_range.array[:] = np.asarray(search_range, dtype=np.int32)
-    _calculate_correlation_error(images, shared_search_range, min_correlation_error, progress)
+    search_range_list = list(search_range)
```

Comment:
I don't think there is a need to make a `list` here. Just use `search_range`

Reply:
mantidimaging\core\rotation\polyfit_correlation.py:60: error: Argument 2 to "_find_shift" has incompatible type "range"; expected "list[int]"  [arg-type]

Reply:
you could change the type annotation for `search_range` in `_find_shift()` back to `range`

Reply:
lol